### DISCUSSION TITLE
TicToc Health

### DISCUSSION THREAD

**Discussion by tonicwave** [CMSgov/price-transparency-guide#668] [Show and tell]
TicToc Health is offering a new set of tools for people working with Transparency in Coverage data.

The Transparency in Coverage rule was issued by the US Department of Health and Human Services (HHS) and aims to increase price transparency and promote competition in the healthcare industry.  The rule requires payers to make clear, accessible, and accurate information about the costs of their healthcare services available to the public in a machine-readable format. 

There are some problems with the MRFs that payers currently offer in their attempts to fulfill their obligations. Complete data is difficult to get from many payers. We explain a little bit about how each payer makes it tricky.

https://TicToc.health has details regarding each payer.

If you are interested in downloading a great deal of payer data yourself, we have collected every URL to every machine-readable-file from all of these payers in a single Table of Contents json file (68MB compressed). You can download this file and pull the individual in-network rate files yourself.

Please let us know if you would like to see specific features in our offerings.

**Comment by dgolden** [comment]
Hi @tonicwave ... this contribution is much appreciated. 

Your directory facilitates the use of our tool which is in beta and free to use:
https://www.healthplanmrf.com/mrfexplorer/mrfreportapp 

To download a CSV report with negotiated rates, copy the URL from your directory into MRF Explorer, select an EIN (optional) and NPIs (recommended to limit the report size) and choose billing codes of interest:

![image](https://user-images.githubusercontent.com/6385204/232092161-038ea548-7cc8-4f2a-bdf2-d451a27d9457.png)

**Comment by riashah28** [comment]
Thanks for sharing - this is great! Do you update your aggregated list of URLs each month as the payer data refreshes?

**Reply by tonicwave** [reply]
Hi!

Yes, we expect to have an update for May, and we currently expect to keep things updated.
